rbi/yast/wfm.rbi

```diff
@@ -0,0 +1,24 @@
+# typed: strong
+module Yast::WFM
+  class NoParameter; end
```

Comment:
this looks quite magic for me. For sure it will need some explanation

Reply:
Hm, you are right, I did perpetrate magic here. Fortunately it did not have time to mutate so I can still remember the reason and will document it.

```rb
#  See Args below
class NoParameter
...
# The following sig is a way to describe an optional parameter which has NOT a default value.
# It is a union of
#  sig { params().returns(T::Array[YCPAny]) }
#  sig { params(i: Integer).returns(YCPAny) }
```